pliers/transformers.py

```diff
@@ -13,6 +13,11 @@
 from copy import deepcopy
 import pandas as pd
 import os
+from functools import partial
```

Comment:
No, good catch. I also just realized my linter wasn't working properly, and fixed it. Will probably make a pass through the entire codebase to clean up other issues like this.